tasks/gulp/compile-assets.js

```diff
@@ -28,6 +28,17 @@ const postcsspseudoclasses = require('postcss-pseudo-classes')({
 // check if destination flag is dist
 const isDist = taskArguments.destination === 'dist' || false
 
+// Set the destination
+const destinationPath = function () {
+  if (taskArguments.destination === 'dist') {
+    return taskArguments.destination
```

Comment:
The return from this and line 36 is the same so maybe they could be combined.

Reply:
Sorry I thought I had updated this but I forgot to push it up. I have merged the two conditions and added a comment.

Reply:
Looks good 👍